frontend/src/metabase/querying/notebook/components/Notebook/Notebook.unit.spec.tsx

```diff
@@ -340,6 +331,9 @@ describe("Notebook", () => {
             data: itemPickerData,
           });
         }
+
+        // Stupid assertion to make linter happy. Assertions are made in convenience functions above
+        expect(true).toBe(true);
```

Comment:
Need to come up with something better here

Reply:
I think this may work, if you put it inside the function:

```
/* eslint-disable jest/expect-expect */
```